### PR TITLE
variable name consistency

### PR DESCRIPTION
**Pull request by nateprewitt**
Just need to clean up a few variable names.

### CODE REVIEW

File: requests/sessions.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def resolve_redirects(self, response, request, stream=False, timeout=None,
         """
 
         redirect_count = 0
-        history = [] # keep track of history
+        hist = [] # keep track of history
```
Comment (Lukasa) [inline requests/sessions.py]: Let's call the other instances `history`, rather than call this `hist`.

Comment (nateprewitt) [reply requests/sessions.py]: :+1:

**Comment by nateprewitt** [comment]
All set.

**Comment by Lukasa** [comment]
So to be clear, the clearer names are on the 3.0 branch already: I just accidentally busted them with names from the master branch like a dummy.

**Comment by nateprewitt** [comment]
Yep @sigmavirus24, `response` and `history` were added to 3.0.0 in commits by Kenneth.

Edit: 66eedec to be specific.

**Comment by jvanasco** [comment]
Is there an ETA of this getting merged in?  I'd like to refile #3871 against the 3.0 branch, but will need to schedule the time to do that after this is merged.

**Comment by nateprewitt** [comment]
I think it's good to go @jvanasco, we were just waiting for confirmation from @kennethreitz. @Lukasa we can probably just merge though since this isn't actually a change, yeah?

**Comment by kennethreitz** [comment]
+1

**Pull request merged**